e2e/process-services/pages/process-details.page.ts

```diff
@@ -42,22 +35,6 @@ export class ProcessDetailsPage {
     completedTask = element(by.css('div[data-automation-id="completed-tasks"]'));
     taskTitle = element(by.css('.adf-activiti-task-details__header'));
 
-    async checkDetailsAreDisplayed(): Promise<void> {
```

Comment:
can't we move this to the processInstanceHeaderPage in this way we could save copy and paste?

Reply:
I've removed the method because i thought it's better to be visible in the test what details exactly we are checking. But i can move it, i don't mind